app/components/post_textbox/index.js

```diff
@@ -42,11 +42,16 @@ function mapStateToProps(state, ownProps) {
     const status = getStatusForUserId(state, currentUserId);
     const userIsOutOfOffice = status === General.OUT_OF_OFFICE;
 
+    const defaultChannel = Object.values(state.entities.channels.channels).find((c) => {
```

Comment:
not a fan of getting this every time the redux store changes, perhaps a selector?

Reply:
Agreed. I'll add a selector.